internal/bminventory/inventory.go

```diff
@@ -4496,7 +4478,8 @@ func (b *bareMetalInventory) UpdateHostIgnitionInternal(ctx context.Context, par
 		return nil, common.NewApiError(http.StatusInternalServerError, err)
 	}
 
-	b.eventsHandler.AddEvent(ctx, params.ClusterID, &params.HostID, models.EventSeverityInfo, fmt.Sprintf("Host %s: custom discovery ignition config was applied", hostutil.GetHostnameForMsg(&h.Host)), time.Now())
+	eventgen.SendHostDiscoveryIgnitionConfigAppliedEvent(ctx, b.eventsHandler, &params.ClusterID, params.HostID,
+		params.ClusterID, hostutil.GetHostnameForMsg(&h.Host))
```

Comment:
the fifth argument should be infra_env_id

Reply:
As mentioned here https://github.com/openshift/assisted-service/pull/2717#discussion_r727761899, since it's a V1 function, we assume here that `infra-env-id = cluster-id`